src/main/java/seedu/duke/item/exercise/FutureExerciseList.java

```diff
@@ -0,0 +1,114 @@
+package seedu.duke.item.exercise;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Comparator;
+
+public class FutureExerciseList extends ExerciseList {
```

Comment:
looks good for now, but i think u will need to change the name of `exerciselist` to `itemlist` later since I am going to remove `exerciselist` attribute in ExerciseList class and move to ItemList class :(